### DISCUSSION TITLE
[Feature] SQLite DB support

### DISCUSSION THREAD

**Discussion by starsoccer** [johanohly/AirTrail#59] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

It would be great to support SQLite or support using local file storage so a postgres database is not required for running AirTrail

**Comment by johanohly** [comment]
Thank you for your feature request!
Supporting any other databases is not in the scope of this project currently, as it would require a substantial reimplementation of key components of the code. 
I'll keep this discussion open for anyone to pitch in with their comments, or to voice their interest.

**Comment by palitu** [comment]
I like the idea of SQLite.  I do not expect that hte DB will get that large, but potentially it is the analytics that will cause some issues?
